lib/core/network/api/aws_amplify/amplify_service.dart

```diff
@@ -226,6 +226,15 @@ class AmplifyService {
     });
   }
 
+  Future<dynamic> removeAvatar() async {
```

Comment:
@n13  I created this, but i am not sure if this service accepts null. Or we should be sending '' when we try to remove the avatar

Reply:
I think I will have to try that out, I have no idea and no documentation. 
In theory it should accept null.